Sources/IO/Geometry/STLWriter/index.js

```diff
@@ -0,0 +1,216 @@
+// Author: Thomas Beznik, with the help of Julien Finet (https://github.com/Kitware/vtk-js/issues/1442)
```

Comment:
@jourdain do we have a more standardized way to cite authors in VTK.js ?

Reply:
https://jsdoc.app/tags-author.html